homeassistant/components/sensor/glances.py

```diff
@@ -40,7 +40,8 @@
     'process_running': ['Running', 'Count'],
     'process_total': ['Total', 'Count'],
     'process_thread': ['Thread', 'Count'],
-    'process_sleeping': ['Sleeping', 'Count']
+    'process_sleeping': ['Sleeping', 'Count'],
+    'cpu_temp': ['CPU Temp', '°'],
```

Comment:
Home Assistant will automatically convert the state into the user preferred units if you return `TEMP_CELSIUS` or `TEMP_FAHRENHEIT` as the unit of measurement (constants defined in `homeassistant.const`)